Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
```

Comment:
I understand your point but the process in order to build the executable is very complex since it depends currently on rebol SDK which is not freely available and actually no more provided. 
I am not involved in the development of the red language but the creator of the language asks me to propose a PR since homebrew is in his opinion the best system to install tools in MacOSX system. 
His point is also to inform that red is an interesting language in the MacOSX world. In some future release, red will only depend on red and it will be possible (I guess) to build red from source. 
Furthermore, as a MacOSX user I do not want to use macport (or other system) to install red. From now, the installation is very basic but Homebrew is also a way to discover some interesting tools.